### PR TITLE
Fixes: C4, flashlight tweakings, and precache and implemented #256

### PR DESCRIPTION
**Pull request by Alf21**

### CODE REVIEW

File: code/gamemode/Game.cs [hunk 1]
```diff
@@ -26,13 +26,17 @@ public partial class Game : Sandbox.Game
...
+        [ConVar.Replicated("ttt_debug")]
         public bool Debug { get; set; } = false;
 
         public Game()
         {
             Instance = this;
 
+            Utils.RNG.Next();
+            Utils.RNG.Next();
+            Utils.RNG.Next();
```
Comment (mzegar) [inline code/gamemode/Game.cs:38]: What is this for?

Comment (Alf21) [reply code/gamemode/Game.cs:38]: Warmup of the RNG, if it's not needed in C#, feel free to remove :)

Comment (mzegar) [reply code/gamemode/Game.cs:38]: nah shouldn't be needed, never actually seen something like this before. Was it done in .lua before?

Comment (Alf21) [reply code/gamemode/Game.cs:38]: yea :D

**Review by mzegar — APPROVED** [review]

**Pull request merged**